site/content/en/docs/contributing/coding-style.md

```diff
@@ -5,5 +5,29 @@ weight: 4
 description: 'Information about JavaScript/Typescript coding style that is used in CVAT development.'
 ---
 
-We use the [Airbnb JavaScript Style Guide](https://github.com/airbnb/javascript) for JavaScript code with a
-little exception - we prefer 4 spaces for indentation of nested blocks and statements.
+## Overview
+
+We use the [Airbnb JavaScript Style Guide](https://github.com/airbnb/javascript) for JavaScript
+code with a little exception - we prefer 4 spaces for indentation of nested blocks and statements.
+
+For Python, we use [Black](https://github.com/psf/black) and
+[isort](https://pycqa.github.io/isort/) to enforce the coding style and autoformat files.
+Currently, not all components implement formatting.
+
+
+## Hooks
+
+It is recommended to use [git pre-commit hooks](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks)
+to avoid the need of manual control of linting and formatting the code. We provide a set of hooks
```

Comment:
Isn't it just one hook (`pre-commit`)?

Reply:
Maybe, what is the problem with this? This one already includes a bunch of calls, and I clearly see it can be extended. I tried writing some custom scripts in the commits above, it turned out to be too complicated to maintain manually. Having everything handled by `list-staged` greatly helps.

Reply:
I'm just talking about the wording here. We only provide one hook (which does multiple things).